src/resources/vue/components/modal.vue

```diff
@@ -0,0 +1,63 @@
+<template>
```

Comment:
Please use 2 tabs instead of 4, this can easily be done using prettier(ctrl+alt+f) it formats the file automatically

Reply:
Please do this for all the files you've changed

Reply:
Please @Huib8

Reply:
Sprich Niederländisch, du verrückter Neonazi